superdesk/publish/publish_content.py

```diff
@@ -51,7 +51,7 @@ def publish():
 
 
 def get_queue_items():
-    lookup = {'state': 'pending', 'destination.delivery_type': {'$ne': 'pull'}}
+    lookup = {'state': {'$in': ['pending', 'in-progress']}, 'destination.delivery_type': {'$ne': 'pull'}}
```

Comment:
Ideally if transmission has failed then the status should be changed to failed instead of keeping it in-progress. Keeping it in-progress the query will bring the items which haven't failed also and might end up transmitting it twice.